src/libs/OptionsListUtils/index.ts

```diff
@@ -688,12 +689,11 @@ function getLastMessageTextForReport({
         // eslint-disable-next-line @typescript-eslint/no-deprecated
         lastMessageTextFromReport = `[${translateLocal('common.attachment')}]`;
     } else if (isModifiedExpenseAction(lastReportAction)) {
-        const properSchemaForModifiedExpenseMessage = getForReportAction({
+        const properSchemaForModifiedExpenseMessage = getForReportActionTemp({
             reportAction: lastReportAction,
-            policyID: report?.policyID,
             movedFromReport,
             movedToReport,
-            policyForMovingExpensesID,
```

Comment:
This key might be used inside the logic of `getForReportAction` which sould be copied to `getForReportActionTemp` as well.